### PR TITLE
Surface event webhooks in the UI on the Settings ➡️ Webhooks page

### PR DESCRIPTION
**Pull request by tinahollygb**
### Features and Changes

Surfaces the Event Webhooks UI on the webhooks page: http://localhost:3000/settings/webhooks

This essentially "launches" the event-based webhooks since it will now be visible in the UI and users will be able to create event-based webhooks.

### Dependencies

- [x] https://github.com/growthbook/growthbook/pull/878

### Testing

- Click "Webhooks" under settings in the sidebar to see.
- Create an event webhook with all currently-available events: `feature.created`, `feature.updated`, `feature.deleted`
-  Create a new feature to test the `feature.created` event
- Delete a feature to test the `feature.deleted` event
- To test `feature.updated`:
  - toggle an environment on/off for a feature
  - put a feature in a project and delete that project. 
  - tag a feature and delete that tag
  - update the description of a feature
  - change the default value of a feature
  - add/remove/change any rules for a feature

### Screenshots

State with event webhooks

image

Empty state

image

### CODE REVIEW

File: packages/front-end/pages/settings/webhooks/index.tsx [hunk 1]
```diff
@@ -29,10 +30,9 @@ const WebhooksPage: FC = () => {
       <h1>SDK Endpoint Webhooks</h1>
       <Webhooks />
 
-      {/* TODO: Include this code when we are ready to launch Event-based Web Hooks */}
-      {/*<div className="mt-5">*/}
-      {/*  <EventWebHooksPage />*/}
-      {/*</div>*/}
+      <div className="mt-5">
+        <EventWebHooksPage />
```
Comment (bryce-fitzsimons) [inline packages/front-end/pages/settings/webhooks/index.tsx:34]: This seems to have some extra padding coming from the `container pagecontents` classes.

Comment (tinahollygb) [reply packages/front-end/pages/settings/webhooks/index.tsx:34]: Good eye! 👁️ I've removed the `container` class which added an extra 8px horizontally. Thanks!

**Review by bryce-fitzsimons — CHANGES_REQUESTED** [review]
- Is it just my local dev? There doesn't seem to be a way to delete or modify these webhooks once they're created.
- There's some odd behavior with the multiselect field. When I choose 1+ events and then click off of the field, the field erases itself (onblur).
- Some UI alignment issues between the SDK Webhooks and Event Webhooks (left versus center CTAs).  Also different font sizes etc (but this is less of an issue if we're going to fast follow)
- Seems like there should be a more descriptive tagline than "Event Webhooks are event-based, and allow you to monitor specific events." Maybe mention a common use case (hooking to your app or 3rd party vendor), as well as a link to the docs?

**Comment by tinahollygb** [comment]
Thanks for the thorough review! Let me address these concerns individually:

> * Is it just my local dev? There doesn't seem to be a way to delete or modify these webhooks once they're created.

This implementation doesn't have editing and deleting capabilities, but it's definitely a good idea to do that before shipping. I had built editing functionality for the modal but it isn't implemented. Deleting has not been implemented on the front-end. I will put this PR in draft until I implement those. Thanks for noticing.

> * There's some odd behavior with the multiselect field. When I choose 1+ events and then click off of the field, the field erases itself (onblur).

This occurs when clicking on the `<label>` field. I've pushed up a change to unnest the label. This field behaves weirdly when it gets selected via label clicking. Rather than solve the problem with this field, I think it may make sense to unnest it like the rest of the app. This has been fixed in 6b8c28cabefd7c8be784df4cb9e2692095ac594a

> * Some UI alignment issues between the SDK Webhooks and Event Webhooks (left versus center CTAs).  Also different font sizes etc (but this is less of an issue if we're going to fast follow)

I'll take another pass at font sizes, didn't notice the differences. As for the alignment, I was going for a card-style empty state like in other parts I've done (e.g. when editing individual portions of the data source). And since the layout is using cards instead of the common table we use everywhere, an empty table state doesn't make sense here. Hopefully when we get a designer we can address things like empty state patterns. I was going to go with this card-style empty state given this section is cards, unless you had another suggestion you think could work better. Please keep in mind, the existing webhooks section will be moving elsewhere so the contrast will look less jarring once that happens.

> * Seems like there should be a more descriptive tagline than "Event Webhooks are event-based, and allow you to monitor specific events." Maybe mention a common use case (hooking to your app or 3rd party vendor), as well as a link to the docs?

I've already gotten the copy reviewed by Jeremy. Feel free to suggest alternate copy, I can replace it if you have something better.

> * If I enter a malformed URL and submit the form, it fails, but there is no visual status for this.

Good catch, I'll need to support server-side errors better. Thanks for testing it.

**Comment by tinahollygb** [comment]
@bryce-fitzsimons your concerns for the following issues have been resolved in [this PR](https://github.com/growthbook/growthbook/pull/878):

- editing a web hook
- deleting a web hook
- showing errors for server errors

It also includes client-side validation. Please take a look whenever you get a moment, thanks!

**Review by bryce-fitzsimons — APPROVED** [review]

**Pull request merged**